pmd-core/pom.xml

```diff
@@ -149,6 +149,11 @@
             <artifactId>checker-qual</artifactId>
             <version>2.5.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.pcollections</groupId>
+            <artifactId>pcollections</artifactId>
```

Comment:
We could do without it in this PR, but I use it in the type resolution code, and I think it's better introduced now rather than later. This can have many many applications